doc: Update the License Activation part of Segger Installation (#6062)
nrfconnect/sdk-nrf: doc/nrf/gs_programming.rst

   .. build_SES_projimport_start

> 4. Click :guilabel:`OK` to add the project to SES. If you are Importing the project for the first time, you need to request and activate a license.

Reviewer: Break to new line after the sentence period.

> 4. Click :guilabel:`OK` to add the project to SES. If you are Importing the project for the first time, you need to request and activate a license.

Reviewer: A bit confused here. In line 43 we mention `Open NCS project`, but in this line suddenly Importing appears, suggesting that some other option is being used or could be used. This can be understood as using "File > Import Project".

> 4. Click :guilabel:`OK` to add the project to SES. If you are Importing the project for the first time, you need to request and activate a license.

Reviewer: Actually, would it make sense to make the part about the license into a subsection? 
```suggestion

First-time project license activation
=========================

If you are adding the project to SES for the first time, you need to request and activate a license.
The following notification appears after you click :guilabel:`OK` to add the project to SES:

<figure>

To request and activate the license, complete the following steps:

<steps a through d>
```

> 

Reviewer: No need for a double blank line.

        .. figure:: images/ses_license_build.png
           :alt: SEGGER Embedded Studio Dashboard notification about missing license

           No commercial-use license detected SES prompt

     a. Click :guilabel:`Activate Your Free License`.
        A request form appears.

     #. Fill in your information and click :guilabel:`Request License`.
        The license is sent to you in an email.

     #. After you receive your license key, click :guilabel:`Enter Activation Key` to activate the license.

     #. Copy-paste the license key and click :guilabel:`Install License`.

>         The license activation window will close. You can now work with the project in the IDE.

Reviewer: ```suggestion
        The license activation window will close.

You can now work with the project in the IDE.
```
The last sentence is the result of the steps, so can go at the end.

   .. build_SES_projimport_start

4. Click :guilabel:`OK` to add the project to SES.

> Activating license for first time

Reviewer 2: ```suggestion
Activating a license for the first time
```

Could we make it more of a full sentence? I think adding in these articles makes it sound more professional.

   .. build_SES_projimport_start

4. Click :guilabel:`OK` to add the project to SES.

> License Activation for first-time use

Reviewer 3: Activation may need to be lowercase here?

> License activation for first-time use
> =====================================

Reviewer: nit: Doesn't this header suggest the license needs to be activated also for later uses?
```suggestion
License activation
==================
```
I think we don't need to repeat "first time" if we state it in the first sentence.

Reviewer: Also, this is a static header, but we're requiring action from the user, so I would go for something like "Activating license".

Author: After an offline discussion with greg - will update this as - Activating license for first time (As the user needs to do the license activation just for the 1st time).

   If you are adding the project for the first time, you need to request and activate a license.
   The following notification appears after you click :guilabel:`OK` to add the project to SES:

        .. figure:: images/ses_license_build.png
           :alt: SEGGER Embedded Studio Dashboard notification about missing license

           No commercial-use license detected SES prompt

   To request and activate the license, complete the following steps:

     a. Click :guilabel:`Activate Your Free License`.
        A request form appears.

> 

Reviewer: Blank lines between steps are not needed.

     #. Fill in your information and click :guilabel:`Request License`.
        The license is sent to you in an email.

     #. After you receive your license key, click :guilabel:`Enter Activation Key` to activate the license.

     #. Copy-paste the license key and click :guilabel:`Install License`.

>         The license activation window closes.

Reviewer: Please fix indentation.

>         The license activation window will close.

Reviewer 2: ```suggestion
        The license activation window closes.
```

   .. build_SES_projimport_start

4. Click :guilabel:`OK` to add the project to SES.
   

> License Activation for First-time   

Reviewer 3: I suggest, "License Activation for first-time use"
